packages/fxa-settings/src/pages/Signin/SigninBounced/index.test.tsx

```diff
@@ -98,12 +79,7 @@ describe('SigninBounced', () => {
   });
 
   it('pushes the user to the /signin page if there is no email address available', () => {
-    renderWithRouter(<SigninBounced emailLookupComplete />);
+    renderWithRouter(<SigninBounced />);
     expect(window.location.href).toBe('/signin');
   });
-
-  it('does not push the user back to the /signin page until the email has finished being checked', () => {
```

Comment:
Is there a reason this test was removed?

Reply:
Same thing as this: https://github.com/mozilla/fxa/pull/15621#discussion_r1287713863